tests/unit/coverage/CoverageSimplifierTest.cpp

```diff
@@ -475,7 +475,7 @@ void object::test<30> ()
         std::vector<std::unique_ptr<Geometry>> result =
         CoverageSimplifier::simplify(input, 10);
     }
-    catch (geos::util::IllegalArgumentException& iae) {
+    catch (geos::util::IllegalArgumentException&) {
```

Comment:
Any reason this change is in this PR?

Reply:
It's only mildly related to MSVC where the primary fix is for, but it is otherwise unrelated. I'm fine to remove it.

Reply:
Probably best to put in a separate PR, to avoid any confusion down the road.